src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/fixtures/statements.fixture.ts

```diff
@@ -0,0 +1,69 @@
+import { assign, merge } from 'lodash';
+import Activity from '../../../../../models/Activity';
+import Agent from '../../../../../models/Agent';
+import InteractionActivityDefinition from '../../../../../models/InteractionActivityDefinition';
+import Statement from '../../../../../models/Statement';
+import SubStatementObject from '../../../../../models/SubStatementObject';
+
+const VERB_ID = 'http://example.org/test-verb';
+
+const statementDefaults: Statement = {
+  id: 'testvalue',
+  authority: {
+    objectType: 'Agent',
+    mbox: 'mailto:authority@test.com',
+  },
+  stored: 'testvalue',
+  timestamp: 'testvalue',
+  version: 'testvalue',
+  actor: {
+    objectType: 'Agent',
+    mbox: 'mailto:actor@test.com',
+  } as Agent,
+  verb: {
+    id: VERB_ID,
+  },
+  object: {
+    objectType: 'Activity',
+    id: 'http://example.org/activity',
+  } as Activity,
+  result: {
+    duration: 'P1Y2M3DT4H5M6S',
+  },
+};
+
+const interactionActivityStatementBase: Statement = merge(
```

Comment:
We should be able to simplify this code in terms of types and readability using the spread operator.